blocks/filter-group/filter-group.css

```diff
@@ -0,0 +1,81 @@
+/* !important added to override system styles to maintain cutoff 
+effect on filter buttons to indicate horizontal scroll */
+.filter-group-wrapper {
+  padding-right: 0 !important;
+
+  @media (min-width: 76rem) {
+    padding: 0 2rem !important;
+  }
+}
+
+.filter-group {
+  margin-block: 1.5rem;
+}
+
+.filter-group__list {
+  display: flex;
+  gap: 0.5rem;
+  overflow-x: auto;
+  padding-block-end: 1rem;
+  padding-inline-start: 0;
+  scrollbar-width: none; /* Firefox */
```

Comment:
did we ask design if they wanted this

Reply:
i mean it looks better

Reply:
This was added just so firefox was consistent with the other browsers, but I can touch base with Jelly.